src/components/Playbutton.jsx

```diff
@@ -1,5 +1,11 @@
-export const Playbutton = () => (
-  <div className="play-button">
+export const Playbutton = ({ handlePlay }) => (
+  <div
+    className="play-button"
+    onClick={() => handlePlay()}
+    onKeyDown={(e) => e.key === "Enter" && handlePlay()}
+    tabIndex={0}
+    role="button"
+  >
```

Comment:
The play-button sits atop another giant button (the video player). It would be good for usability to give it some sort of hover, active & focus-visible effects.